src/Control/Effect.hs

```diff
@@ -19,3 +20,6 @@ import Control.Effect.State     as X (State, StateC)
 import Control.Effect.Sum       as X ((:+:), Member, send)
 import Control.Effect.Trace     as X (Trace, TraceByPrintingC, TraceByIgnoringC, TraceByReturningC)
 import Control.Effect.Writer    as X (Writer, WriterC)
+
+type Eff m = m
```

Comment:
There’s no `Internal` module anymore, so I’m open to suggestions as to where to place this until we kick it out in 0.4.0.

Reply:
This is a fine place for it 👍 Further, I think this is a really good definition to use for it in the interim.